src/main/webapp/js/addrole.js

```diff
@@ -100,10 +100,10 @@ const sendPostRequest = (postUrl, json) => {
 
     xhr.onload = () => {
         if (xhr.status === 200) {
-            alert('The role was added successfully');
-            location.reload(); // refresh the page
+            showNotificationOK('The role was added successfully');
+            //location.reload(); // refresh the page
```

Comment:
we shouldn't leave commented out code in, it looks like the function handles this?

```suggestion
```

Reply:
I forgot that, removed now